### PR TITLE
Add 1.10 support

### LINKED ISSUE #51: Support Diablo II version 1.10
Diablo II version 1.10 is a relatively popular version for developing and releasing mods for. The expectation for supporting 1.10 is to port over the 1.09D patches to 1.10, and to sort out any differences between the two. All Diablo II prerequisites in SGD2MAPI have already been defined.

### PR DESCRIPTION
**Pull request by Jarcho**
fixes #51

### CODE REVIEW

**Comment by IAmTrial** [comment]
I think you meant to apply the changes to master branch.

**Comment by IAmTrial** [comment]
I have changed the base branch, and can see the changes you intended to make. I'll review it some time within the next two weeks for correctness.

**Comment by Jarcho** [comment]
Fixed a typo. 'B' and '8' look too similar. Should be functioning properly now.

**Comment by Jarcho** [comment]
Turns out this still has an error. The game crashes when opening the either of the right panels on 800x600 mode. All other resolutions appear to work, and everything else on 800x600 also appears to work.

**Comment by IAmTrial** [comment]
I have been a bit busy, I will try to review it as soon as I can.

**Comment by Jarcho** [comment]
Right panel on 800x600 has been fixed. Hopefully that's the last issue.

File: SlashGaming-Diablo-II-Free-Resolution/src/patches/interface_bar/d2client_enable_800_new_stats_button_patch/d2client_enable_800_new_stats_button_patch_1_09d.cc [hunk 1]
```diff
@@ -470,6 +580,16 @@ Enable800NewStatsButtonPatch_1_09D::GetPatchAddressAndSize12() {
           0x48B81 - 0x48B5C
       );
     }
+
+    case ::d2::GameVersion::k1_10: {
+      return PatchAddressAndSize(
+          ::mapi::GameAddress::FromOffset(
+              ::d2::DefaultLibrary::kD2Client,
+              0x4F5E7
```
Comment (IAmTrial) [inline SlashGaming-Diablo-II-Free-Resolution/src/patches/interface_bar/d2client_enable_800_new_stats_button_patch/d2client_enable_800_new_stats_button_patch_1_09d.cc:588]: While this patch location starts two instructions up, it ends on the same relative places, and performs the same thing.

File: SlashGaming-Diablo-II-Free-Resolution/src/patches/required/d2gfx_set_display_width_and_height_patch/d2gfx_set_display_width_and_height_patch_1_09d.cc [hunk 2]
```diff
@@ -84,6 +84,16 @@ SetDisplayWidthAndHeightPatch_1_09D::GetPatchAddressAndSize01() {
...
       );
     }
+
+    case ::d2::GameVersion::k1_10: {
+      return PatchAddressAndSize(
+          ::mapi::GameAddress::FromOffset(
+              ::d2::DefaultLibrary::kD2GFX,
+              0x4B50
+          ),
+          0x4B89 - 0x4B50
```
Comment (IAmTrial) [inline SlashGaming-Diablo-II-Free-Resolution/src/patches/required/d2gfx_set_display_width_and_height_patch/d2gfx_set_display_width_and_height_patch_1_09d.cc:94]: Change this ending of the patch to 0x4B88.

**Review by IAmTrial — CHANGES_REQUESTED** [review]

**Comment by IAmTrial** [comment]
Everything seems to be mostly correct, except for one bug that crashes the game. Once that is fixed, I will approve the changes and merge the PR.

**Comment by DoctorGoat** [comment]
Hooray! I'm pretty excited about this, thank you for the work

File: SlashGaming-Diablo-II-Free-Resolution/src/patches/required/d2gfx_set_display_width_and_height_patch/d2gfx_set_display_width_and_height_patch_1_09d.cc (hunk 2 above)
Comment (Jarcho) [reply SlashGaming-Diablo-II-Free-Resolution/src/patches/required/d2gfx_set_display_width_and_height_patch/d2gfx_set_display_width_and_height_patch_1_09d.cc]: Done

**Review by IAmTrial — APPROVED** [review]

**Pull request merged**